src/App.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { useEffect, useState } from 'react';
-import { NuiProvider } from 'react-fivem-hooks';
-import { useHistory } from 'react-router-dom';
+import { NuiProvider } from 'fivem-nui-react-lib';
```

Comment:
You don't actually need this. Just use the `useNuiHook` from npwd (copy-paste) into the codebase. No need for the lib here.